### ISSUE TITLE
RFE: possibility to change category of a message in config

### ISSUE THREAD

**Issue by scop** [PyCQA/pylint#3251]
### Is your feature request related to a problem? Please describe

As discussed in #3250, an option to change the category of a message would be useful.

My first use case would be to exit with a nonzero status if some (currently) informational messages are emitted, e.g. `useless-suppression` and `use-symbolic-message-instead`.

A secondary use case I can think of would be to lower the severity of a warning or an error message to informational. The difference of this compared to disabling messages is that the messages would be displayed in the output, but not cause a nonzero exit.

### Describe the solution you'd like

Configurabie categories/severity levels for messages, including their effect on the exit status.

### Additional context

I gather the categories are currently kind of hardwired to the messages identifiers, so maybe it would be good to not change them but just their "effect" so there's no confusion about what is being referred to in online discussions etc.

**Comment by daoiqi** [comment]
Yes, In something I want to change message level to warning. so I can get precise `evaluation` score

```python
b = 1,
```

R1707: Disallow trailing comma tuple (trailing-comma-tuple)
